projects/ui-components/lf-metadata/field-components/field-base-parts/lf-field-base/lf-field-base/lf-field-base.component.css

```diff
@@ -19,86 +20,170 @@
   width: 100%;
 }
 
-::ng-deep ngx-mat-timepicker > form > table > tbody > tr > td > mat-form-field.mat-form-field-appearance-legacy > .mat-form-field-wrapper {
+::ng-deep ngx-mat-timepicker > form > table > tbody > tr > td > mat-form-field.mat-mdc-form-field-appearance-legacy > .mat-mdc-text-field-wrapper {
   padding: 1em 0;
 }
 
-::ng-deep ngx-mat-timepicker > form > table > tbody > tr > td > mat-form-field.mat-form-field-appearance-legacy > .mat-form-field-wrapper > .mat-form-field-flex > .mat-form-field-infix {
+::ng-deep ngx-mat-timepicker > form > table > tbody > tr > td > mat-form-field.mat-mdc-form-field-appearance-legacy > .mat-mdc-text-field-wrapper > .mat-mdc-form-field-flex > .mat-mdc-form-field-infix {

```

Comment:
review styles in this file, include lf style with each (I know there wasn't before, it should be okay but best practices to make it specific)

Reply:
I think we can remove this style